roles/openshift_cli/templates/openshift.j2

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+if [ ! -d ~/.kube ]; then
+   mkdir -m 0700 ~/.kube
+fi
+cmd=`basename $0`
+user=`id -u`
+group=`id -g`
+
+docker run -it --user=${user}:${group} -v ~/.kube:/root/.kube:z -e KUBECONFIG=/root/.kube/config --entrypoint ${cmd} --rm {{ openshift.common.cli_image }} ${@}
```

Comment:
will that work for users neq root? If not kubeconfig might be mounted in another place.